src/config/dark-sites.config

```diff
@@ -519,6 +519,8 @@ json-diff.com
 justfive.news
 kadantiscam.netlify.app
 kaoskrew.org
+karol.gay
+karolpeszek.com
```

Comment:
CNAME/redirections aren't allowed to be included within this site list. Please remove.